core/server/data/schema/fixtures/fixtures.json

```diff
@@ -96,6 +100,11 @@
                     "action_type": "send",
                     "object_type": "mail"
                 },
+                {
+                    "name": "Schedule content",
+                    "action_type": "schedule",
+                    "object_type": "schedule"
```

Comment:
Should this object type be `post` and possibly another one for `page`?

Reply:
I'm wondering if this permission should just be "publish" "post" :thinking:

Reply:
> Should this object type be post and possibly another one for page?

Was thinking about scheduling in more generic terms of scheduling a "resource" or "content". 

> I'm wondering if this permission should just be "publish" "post"

My first response was: "nah this is different" :sweat_smile: But after looking at it from the higher point all aw allow to do is "publish" here so it :100: makes sense! The only bit I'm not sure about is the "post"/"page" bit. It's a generic action on anything that is "publishable" and don't see the point of making it that much granular :thinking: On the other hand why invent a new resource type?

After a second look at this, think naming this permission might've been:
```
{
   "name": "Publish resource",
   "action_type": "publish",
   "object_type": "publishable"
}
OR
{
   "name": "Publish  post/page",
   "action_type": "publish",
   "object_type": "post" / "page"
}
```

Any other ideas? The second approach with "post" / "page" would probably need some more customizations in the "frame" as we only can have one resource per controller at the moment (or we introduce 2 separate controllers :thinking:)

Reply:
@allouis @kevinansfield @rishabhgrg ping (what we talked about on the call)

Reply:
FWIW I don't think this is a blocker.

We don't have a `page` object type - and I don't think we need to introduce one.

The permission here seems to be specifically `publish: post`. By renaming it, we don't change any functionality, but we set some groundwork for fixing the post permissions in future.

As for the scheduler having more permissions in future - this is totally doable, as the scheduler will have the scheduler role - which can have many permissions.

We actively _want_ permissions  to be more granular, in the future, starting now isn't a terrible idea IMO.

Again I don't want to block on this - get it merged whatever which way - permissions and roles _are_ modifiable in the future

Reply:
🤔 If I understand the permissions correct, the object here is meant to give access to `action_type` method on `object_type` controller, I think changing the method on `schedules` controller to `publish` instead `schedule`(and the `action_type` here in turn) is a nice middle ground which also keeps the flexibility or control of how to publish with scheduler and allow it to be more custom than tying it with `post` controller directly. Could be off direction here if i am missing some permissions internal working 😬

Reply:
Permissions are not based on controller methods, they're based on `action`s on `object`s which we'd like to restrict to particular users/actors.

IMO we should name them accordingly

Reply:
Cool, makes sense in that case, will just need minor changes to permissions file to handle it if we go ahead, will leave it on @gargol's call if its part of this PR :)

Reply:
The messed up thing about permissions is that controllers depend on them in a way that `action` and `object` are being matched with `docName` and the controller method that is being called. Will try to go with `publish` and `post` as new permission for scheduler integration, but if it ends up a bit PITA going with what is there already: `schedule` / `schedule`.

Reply:
@allouis @rishabhgrg Note, the side effect and **behavior change** that is connected with going with `publish`/`post` combination is that other roles will start having access to this endpoint (previously only scheduler had the right to publish).

From a functional perspective, it doesn't change much, because every permission that had `post: "all"` already could "publish" through edit endpoint. But still thought something worth pointing out.

Reply:
> From a functional perspective, it doesn't change much, because every permission that had post: "all" already could "publish" through edit endpoint. But still thought something worth pointing out.
Yeah its definitely important to keep it in mind and some point later make a note of it to also distinguish it with `edit`, but looks good for now